### DISCUSSION TITLE
High CPU usage using @21hsmw nodriver support

### DISCUSSION THREAD

**Discussion by Flameeeeee** [FlareSolverr/FlareSolverr#1376] [General]
Hi @21hsmw 

I am using your fix since a few days and it indeed fixed the flaresolverr issue. 
However, since setting it up, I'm experiencing alarming CPU usage on my server. 

![image](https://github.com/user-attachments/assets/1fcca183-56b2-4ff8-9152-8bbe2a54e24e)
![image](https://github.com/user-attachments/assets/580b013c-474d-427f-900f-fad92c0ec890)

Do you have any idea what might be going on ? The only time the CPU usage decreases is when the container is restarted. 
It then ramps up to reach **100% CPU usage** in the next hours. 

I am quite surprised to be the only one noting this behavior (I may have miss some messages). 

Thanks for your support.

**Comment by Flameeeeee** [comment]
I limited the container to a single CPU but the PSI is still locked at 100%
![image](https://github.com/user-attachments/assets/4e556be6-6ecc-42ae-9e74-e749d0a6c2d1)

**Comment by bast1212** [comment]
I have the same problem

**Comment by ilike2burnthing** [comment]
Report on relevant PR
